etc/spack/defaults/concretizer.yaml

```diff
@@ -10,6 +10,9 @@
 # or by adding a `concretizer:` section to an environment.
 # -------------------------------------------------------------------------
 concretizer:
+  # Maximum time, in seconds, allowed to clingo for the 'solve' phase. If set
+  # to -1, there is no time limit.
+  timeout: -1
```

Comment:
Isn't it more traditional (even in [other places](https://github.com/spack/spack/blob/627544191aa7f8944cd883c220beb7efeed44b51/etc/spack/defaults/config.yaml#L95) in spack) to use `0` to indicate no timeout?

Reply:
We could do it, but then I have to translate to clingo API, that uses 0 to probe the state (0 seconds)

Reply:
See https://github.com/spack/spack/pull/47661/commits/734eee0ffbe932858d4b36b2b5fadf49efaeb426